cmd/cogito/main_test.go

```diff
@@ -63,6 +63,10 @@ func TestRunPutSuccess(t *testing.T) {
 	var ghReq github.AddRequest
 	var ghUrl *url.URL
 	gitHubSpy := testhelp.SpyHttpServer(&ghReq, nil, &ghUrl, http.StatusCreated)
+	gitHubSpyURL, err := url.Parse(gitHubSpy.URL)
+	if err != nil {
+		t.Fatalf("error parsing SpyHttpServer URL: %s", err)
+	}
```

Comment:
we can `assert.NilError` here as usual

Reply:
Fixed in: https://github.com/Pix4D/cogito/commit/66f1cf52187b4780f4fb0775009bab43f2a2155d